src/main/resources/templates/registration_page.html

```diff
@@ -22,18 +22,21 @@
 <body>
 <main>
     <h1 class="mb-4 text-center">Links</h1>
-    <form th:action="@{/register}" method="post" enctype="application/x-www-form-urlencoded" autocomplete="on">
+    <form th:action="@{/register}" th:object="${user}" method="post" enctype="application/x-www-form-urlencoded" autocomplete="on">
         <div class="mb-3">
             <label for="username" class="form-label">Username</label>
-            <input type="text" class="form-control" id="username" aria-describedby="emailHelp" name="username">
+            <input type="text" class="form-control" id="username" aria-describedby="emailHelp" name="username" th:field="*{username}">
```

Comment:
There is a bug here - the values, entered by a user, are lost in case of validation error. We need to fix this.

Steps to reproduce:

1. Go to registration page.
2. Enter email and password only (the username should be empty).
3. Click 'Register' button.
4. Bug: the email and password are empty.

Expected result: the email and password should remain the same as before clicking at 'Register' button.

Reply:
Found the solution, but think that should apply it to fields Username and Email. Not Password.